src/meta/expressionEvaluator/functions/validatorNotEmpty.ts

```diff
@@ -9,11 +9,19 @@ export const validatorNotEmpty: ExpressionFunction<Context> = {
   name: 'validatorNotEmpty',
   minArity: 1,
   executor: () => {
-    return (value?: string): NodeValueValidation => {
-      const isEmpty = Objects.isEmpty(value)
-      const valid = !isEmpty // Invert the result of isEmpty
+    return (value?: string, decimalValues?: Array<number>): NodeValueValidation => {
```

Comment:
The condition should be in plain js , not here . In the formula itself .

The formulas should look like : 

varA || varB ? ValidatorNotEmpty(...) : null .

Does it make sense ?